test/hotspot/jtreg/serviceability/jvmti/SuspendWithObjectMonitorWait/SuspendWithObjectMonitorWait.java

```diff
@@ -121,11 +137,11 @@
 // <launch returns>     waiter running
 // launch resumer       enter threadLock
 // <launch returns>     while !READY_TO_NOTIFY  resumer running
-// delay 1-second         threadLock.wait(1)    wait for notify
+// :                    threadLock.wait(1)      wait for notify
```

Comment:
The wait(1) call is in the while-loop. Please insert two spaces before the `threadlock.wait(1)` call.